### ISSUE TITLE
Not an issue [SCREENREADER]: Labels SHOULD only be read once per input group

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9230]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

```diff
! This might be a sitewide issue, related to the forms system.
! Caregiver team, let me know if this is due to the forms system, and I will reassign the ticket.
```

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Labels for radio input groups should only be read once per group. Currently, the labels for the inputs are repeated for screen readers. **[Here is a recording](https://dsva.slack.com/archives/C8E985R32/p1589836664111600?thread_ts=1589835592.111000&cid=C8E985R32)** of the repeated labels.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to read the content of a page without unnecessary repetition of content.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/caregivers/ in browser
2. Start screenreading device listed in Environment
3. Navigate to the section on Health care coverage
4. Verify that the radio input group labels are repeated

## Possible Fixes (optional)

- Adjust the design so that the 'Learn more about [insurance type]' show/hide component follows the radio inputs (so that the group label may be paired with the associated form elements, without manipulating the DOM).

```diff
<form class="rjsf" novalidate="">
  <div>
    <div class="input-section schemaform-field-container">
      <div class="schemaform-block-header">
        <h5 class="vads-u-font-size--h4">Health care coverage</h5>
        <div class="vads-u-margin-bottom--4 vads-u-margin-top--2">
          <p>Please complete the following information about the Primary Family Caregiver’s health coverage.</p>
        </div>
      </div>
      <div>
        <fieldset class="schemaform-field-template schemaform-first-field">
          <legend class="schemaform-label" id="root_primaryMedicaidEnrolled-label">Is the Primary Family Caregiver currently in enrolled in Medicaid?<span class="schemaform-required-span">(*Required)</span></legend>
          <div class="vads-u-margin-y--1p5 vads-u-margin-bottom--2p5">
            <div class="form-expanding-group">
              <button aria-controls="tooltip-5" aria-expanded="false" class="additional-info-button va-button-link" type="button"><span class="additional-info-title">Learn more about Medicaid<i class="fas fa-angle-down"></i></span></button><span id="tooltip-5"></span>
            </div>
          </div>
          <div class="schemaform-widget-wrapper">
            <div class="form-radio-buttons">
              <input id="root_primaryMedicaidEnrolledYes" name="root_primaryMedicaidEnrolled" type="radio" value="Y"><label for="root_primaryMedicaidEnrolledYes">Yes</label><input id="root_primaryMedicaidEnrolledNo" name="root_primaryMedicaidEnrolled" type="radio" value="N"><label for="root_primaryMedicaidEnrolledNo">No</label>
            </div>
          </div>
          <div></div>
        </fieldset>
      </div>
      <div>
        <fieldset class="schemaform-field-template">
          <legend class="schemaform-label" id="root_primaryMedicareEnrolled-label">Is the Primary Family Caregiver currently in enrolled in Medicare?<span class="schemaform-required-span">(*Required)</span></legend>
          <div class="vads-u-margin-y--1p5 vads-u-margin-bottom--2p5">
            <div class="form-expanding-group">
              <button aria-controls="tooltip-6" aria-expanded="false" class="additional-info-button va-button-link" type="button"><span class="additional-info-title">Learn more about Medicare<i class="fas fa-angle-down"></i></span></button><span id="tooltip-6"></span>
            </div>
          </div>
          <div class="schemaform-widget-wrapper">
            <div class="form-radio-buttons">
              <input id="root_primaryMedicareEnrolledYes" name="root_primaryMedicareEnrolled" type="radio" value="Y"><label for="root_primaryMedicareEnrolledYes">Yes</label><input id="root_primaryMedicareEnrolledNo" name="root_primaryMedicareEnrolled" type="radio" value="N"><label for="root_primaryMedicareEnrolledNo">No</label>
            </div>
          </div>
          <div></div>
        </fieldset>
      </div>
      <div>
        <fieldset class="schemaform-field-template">
          <legend class="schemaform-label" id="root_primaryTricareEnrolled-label">Is the Primary Family Caregiver currently in enrolled in Tricare?<span class="schemaform-required-span">(*Required)</span></legend>
          <div class="vads-u-margin-y--1p5 vads-u-margin-bottom--2p5">
            <div class="form-expanding-group">
              <button aria-controls="tooltip-7" aria-expanded="false" class="additional-info-button va-button-link" type="button"><span class="additional-info-title">Learn more about Tricare<i class="fas fa-angle-down"></i></span></button><span id="tooltip-7"></span>
            </div>
          </div>
          <div class="schemaform-widget-wrapper">
            <div class="form-radio-buttons">
              <input id="root_primaryTricareEnrolledYes" name="root_primaryTricareEnrolled" type="radio" value="Y"><label for="root_primaryTricareEnrolledYes">Yes</label><input id="root_primaryTricareEnrolledNo" name="root_primaryTricareEnrolled" type="radio" value="N"><label for="root_primaryTricareEnrolledNo">No</label>
            </div>
          </div>
          <div></div>
        </fieldset>
      </div>
      <div>
        <fieldset class="schemaform-field-template">
          <legend class="schemaform-label" id="root_primaryChampvaEnrolled-label">Is the Primary Family Caregiver currently in enrolled in CHAMPVA?<span class="schemaform-required-span">(*Required)</span></legend>
          <div class="vads-u-margin-y--1p5 vads-u-margin-bottom--2p5">
            <div class="form-expanding-group">
              <button aria-controls="tooltip-8" aria-expanded="false" class="additional-info-button va-button-link" type="button"><span class="additional-info-title">Learn more about CHAMPVA<i class="fas fa-angle-down"></i></span></button><span id="tooltip-8"></span>
            </div>
          </div>
          <div class="schemaform-widget-wrapper">
            <div class="form-radio-buttons">
              <input id="root_primaryChampvaEnrolledYes" name="root_primaryChampvaEnrolled" type="radio" value="Y"><label for="root_primaryChampvaEnrolledYes">Yes</label><input id="root_primaryChampvaEnrolledNo" name="root_primaryChampvaEnrolled" type="radio" value="N"><label for="root_primaryChampvaEnrolledNo">No</label>
            </div>
          </div>
          <div></div>
        </fieldset>
      </div>
      <div class="form-expanding-group">
        <div>
          <fieldset class="schemaform-field-template">
            <legend class="schemaform-label" id="root_primaryOtherHealthInsurance-label">Is the Primary Family Caregiver currently enrolled in other health care insurance?<span class="schemaform-required-span">(*Required)</span></legend>
            <div class="schemaform-widget-wrapper">
              <div class="form-radio-buttons">
                <input id="root_primaryOtherHealthInsuranceYes" name="root_primaryOtherHealthInsurance" type="radio" value="Y"><label for="root_primaryOtherHealthInsuranceYes">Yes</label><input id="root_primaryOtherHealthInsuranceNo" name="root_primaryOtherHealthInsurance" type="radio" value="N"><label for="root_primaryOtherHealthInsuranceNo">No</label>
              </div>
            </div>
            <div></div>
          </fieldset>
        </div><span></span>
      </div>
    </div>
  </div>
  <div class="row form-progress-buttons schemaform-buttons">
    <div class="small-6 medium-5 columns">
      <button class="usa-button-secondary null" id="2-continueButton" type="button"><span class="button-icon">«</span> Back</button>
    </div>
    <div class="small-6 medium-5 end columns">
      <button class="usa-button-primary null" id="3-continueButton" type="submit">Continue <span class="button-icon">»</span></button>
    </div>
  </div>
  <div>
    <div></div>
  </div>
</form>
```

## WCAG or Vendor Guidance (optional)

* [W3C WCAG Info and Relationships: Understanding SC 1.3.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html)
